content/docs/setup/kubernetes/quick-start.md

```diff
@@ -268,11 +268,19 @@ customize Istio to your needs.
 
     _**OR**_
 
-    b)  [Render Kubernetes manifest with Helm and deploy with kubectl](/docs/setup/kubernetes/helm-install/#option-1-install-with-helm-via-helm-template).
+    b) Install Istio and enable mutual TLS authenticaiton globally between sidecars. This option is only recommended for new clusters, and all applications are installed with sidecars. Otherwise, we recommend to choose option a) and incrementally adopt Istio mutual TLS later on.
```

Comment:
> incrementally adopt Istio mutual TLS later on.

Should we link to where @diemtvu 's doc about per-service enabling of mTLS by using DR and Authn Policy?